### DISCUSSION TITLE
Manually start the circular pump to reduce buffer temperature

### DISCUSSION THREAD

**Discussion by rechi2** [open3e/open3e#109] [Q&A]
Hi there.
I think a few topics where addressed already in https://github.com/open3e/open3e/discussions/91.

However, in my setup, I have a vitocal and a vitopalor, where the vitopalor is supposed to run 24/7 (except regeneration). The thermal output of the VP is going into the buffer, which is also heated by the Vcal and supplies the high temperature wall heating of the house.
My issue in summer time is that the VP is powering up the buffer and the fuell cell shuts down at 50C return temperatur from the buffer. I want the VP to run as much as possible, not shutting down due to high temperatur of the return.
My idea is that if the buffer (or the fuel cell return temp) goes anywhere near 47C, I would force the circular pump to start withdrawing temperature (energy) and to reduce the temperatur of the buffer.

But I am not sure where to start. I would like to ensure that VCAL doesn't power up the heat pump, only the pump circulating the buffer with the wall heating. Can I do that by setting the values of
988_MixerTwoCircuitFlowTemperatureTargetSetpoint 
or 
1644_MixerTwoCircuitCurrentTemperatureSetpoint
or do I have to set any other values?

Would appreciate any hint where to start looking at.

Thanks!
Thomas

**Comment by Philip-Wiege** [comment]
Hi Thomas,

I dont know how to manually trigger the circulator pump of the vitocal but maybe there is a workaround:

Set the vitocal to heating in the app with a very low heating curve. Then the circulator pump should come on but the vitocal should not start. If it is very warm outside, the summer saving or energy saving function might stop the circulator pump. Then we have to see if you can disable the saving functions.

If you can set the vitocal to heating mode via open3e you can trigger this automatically at some buffer temperature if you have some kind of home automation.

BTW: Why do you need the vitocal's circulator pump? Does the Vitocalor and its buffer not have any circulator pump for heating?

Best regards
Philip

Wenn auf Deutsch, einfach auf Deutsch weitermachen

**Reply by rechi2** [reply]
Hi Philip,

danke für die Antwort. Ich nutze Homeassistant und danke für den Tipp das Heizen einfach einzuschalten.
Der Vitovalor nutzt den gleiche Puffer wie die VCAL, sind aber nicht miteinander steuerungstechnisch gekoppelt. Deshalb muss ich da intervenieren wenn der Puffer sich zu stark aufheizt durch den Vitovalor. 
Ich versuch am Wochenende mal per homeassistant Daten auf die Heizung zu schreiben. (an-/ausschalten). Dann könnte ich einfach die Puffertemp überwachen und z.B. bei 47° die Heizung einschalten bis der Puffer auf unter 40 grad fällt. Die flache Heizkurve kann ich ja manuell einstellen anfangs.
LG
Thomas

**Reply by Philip-Wiege** [reply]
Hi Thomas,

das klingt doch nach nem guten Ansatz. Die Heizkurve müsste man sogar per open3e schreiben können. Dann könntest du das aktivieren der Heizung und das setzen der flachen Heizkurve mit viel negativem Offset automatisieren. Wenn deine Schlussbedingung erreicht ist, könntest du dann ja die Heizkurve wieder zurücksetzen.

Ich habe selber noch nie die Heizung per open3e aktiviert. Ich gehe davon aus, dass man das Zeitprogramm als einen Datenpunkt schreiben kann. Zusätzlich müsste es eigentlich noch nen Datenpunkt mit der Einstellung des Heizmodus geben z.b. aus, Heizen, Kühlen, Heizen / Kühlen

Grüße
Philip

**Reply by Wolfgang-03** [reply]
Hi Thomas, 

meiner Erfahrung nach, mit der Vitocal 250, musst Du nur den Heizkreis 1 (HK1) einschalten

```1415_mixeronecircuitoperationstate```

auf "Reduziert" sollte dabei reichen und die Temperatur vom Heizkreis, kannst Du auch beliebig nach unten stellen, 3 Grad ist glaube ich der niedrigste Wert der akzeptiert wird. Bei mir ist es so, dass die Zirkulationspumpe immer (24/7) läuft, wenn HK1 nicht auf "Abschaltbetrieb" steht. 

Viele Grüße
Wolfgang
